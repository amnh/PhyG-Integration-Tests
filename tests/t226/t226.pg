--fuse(spr:5), inputting fastc files along with tcm
set(seed:1634561640)
read(rename: "woman_rename.txt")
read(fastc:"woman.fastc", tcm: "woman-11.tcm")
read(fastc:"woman_cox1_spaced.fastc")
read(include:"Include_woman16.txt")
read(newick:"../t80/t80.tre")
fuse(pairs:5, spr:5)
report("t226.csv", data, overwrite)
report("t226.tre", newick, graphs, overwrite)