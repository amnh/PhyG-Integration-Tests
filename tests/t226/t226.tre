(AlbanianTosk:68.0,(ArmenianWestern:30.0,((Scots:35.0,(Assamese:28.0,SanskritVedic:19.0)HTU23:32.0)HTU24:14.0,((((Cornish:29.0,Manx:21.0)HTU19:17.0,(Danish:26.5,(OldPrussian:30.0,(Faroese:25.5,Icelandic:58.5)HTU17:24.0)HTU18:11.5)HTU20:12.5)HTU21:34.0,(AncientIonicGreek:45.0,Irish:29.0)HTU22:13.0)HTU25:15.5,(Yiddish:31.0,(EasternFrisianSaterland:34.5,KolschColognian:46.5)HTU30:16.5)HTU28:18.0)HTU29:16.5)HTU26:11.5)HTU27:41.0)HTU16[728.0];
(AlbanianTosk:76.5,(Scots:29.0,(ArmenianWestern:27.0,((Irish:30.5,(AncientIonicGreek:53.5,((Cornish:29.0,Manx:21.0)HTU29:17.0,(Danish:28.0,(OldPrussian:30.5,(Faroese:26.5,Icelandic:57.5)HTU27:25.0)HTU28:14.0)HTU30:15.5)HTU25:13.5)HTU26:13.5)HTU20:21.0,((Assamese:30.5,SanskritVedic:15.0)HTU18:28.5,(Yiddish:30.0,(EasternFrisianSaterland:36.0,KolschColognian:45.0)HTU17:15.5)HTU19:22.5)HTU21:15.0)HTU22:13.0)HTU23:20.0)HTU24:59.0)HTU16[727.0];
(AlbanianTosk:68.0,(Scots:29.5,(ArmenianWestern:25.5,((Assamese:30.0,SanskritVedic:15.5)HTU20:29.0,((Yiddish:31.5,(EasternFrisianSaterland:35.5,KolschColognian:45.5)HTU17:15.0)HTU19:18.0,((AncientIonicGreek:44.5,Irish:29.5)HTU18:24.0,((Cornish:27.0,Manx:23.0)HTU29:21.5,(Danish:27.0,(OldPrussian:30.0,(Faroese:26.0,Icelandic:58.0)HTU27:23.0)HTU28:12.0)HTU30:15.5)HTU25:17.0)HTU26:19.0)HTU21:18.5)HTU22:12.5)HTU23:20.5)HTU24:49.5)HTU16[721.0];
(AlbanianTosk:69.0,(Scots:29.0,(ArmenianWestern:25.0,((Assamese:30.5,SanskritVedic:15.0)HTU18:27.5,(Yiddish:35.0,(KolschColognian:45.0,(EasternFrisianSaterland:29.5,((AncientIonicGreek:45.5,Irish:28.5)HTU29:25.0,((Cornish:29.5,Manx:20.5)HTU27:19.5,(Danish:26.0,(OldPrussian:31.0,(Faroese:26.0,Icelandic:58.0)HTU25:23.5)HTU26:12.5)HTU28:14.5)HTU30:17.5)HTU23:17.5)HTU24:21.5)HTU17:14.5)HTU19:23.0)HTU20:14.5)HTU21:21.5)HTU22:53.0)HTU16[720.0];
(AlbanianTosk:67.0,((ArmenianWestern:25.0,(Assamese:30.5,SanskritVedic:15.0)HTU17:28.0)HTU18:22.5,(Scots:28.5,((((Cornish:26.0,Manx:24.0)HTU25:23.0,(Danish:27.0,(OldPrussian:27.5,(Faroese:39.5,Icelandic:53.0)HTU23:19.5)HTU24:14.0)HTU26:17.5)HTU27:19.5,(AncientIonicGreek:42.5,Irish:31.5)HTU28:13.0)HTU29:17.5,(Yiddish:30.5,(EasternFrisianSaterland:33.5,KolschColognian:47.5)HTU22:16.0)HTU30:19.0)HTU20:18.5)HTU21:17.0)HTU19:50.5)HTU16[717.0];
(AlbanianTosk:67.5,(Scots:30.5,((ArmenianWestern:23.0,(Assamese:30.5,SanskritVedic:15.0)HTU17:28.5)HTU18:15.5,((((Cornish:29.0,Manx:21.0)HTU25:17.0,(Danish:27.0,(OldPrussian:29.5,(Faroese:26.0,Icelandic:58.0)HTU23:23.5)HTU24:11.5)HTU26:13.0)HTU27:32.5,(AncientIonicGreek:46.5,Irish:27.5)HTU28:14.5)HTU29:16.0,(Yiddish:30.5,(EasternFrisianSaterland:34.5,KolschColognian:46.5)HTU22:18.0)HTU30:18.0)HTU20:15.0)HTU21:17.0)HTU19:43.5)HTU16[717.0];
(AlbanianTosk:67.0,((ArmenianWestern:25.0,(Assamese:30.5,SanskritVedic:15.0)HTU19:28.0)HTU20:22.5,(Scots:28.5,((Yiddish:30.5,(EasternFrisianSaterland:33.5,KolschColognian:47.5)HTU17:16.0)HTU18:19.0,((AncientIonicGreek:42.5,Irish:31.5)HTU29:13.0,((Cornish:26.0,Manx:24.0)HTU27:23.0,(Danish:27.0,(OldPrussian:27.5,(Faroese:39.5,Icelandic:53.0)HTU25:19.5)HTU26:14.0)HTU28:17.5)HTU30:19.5)HTU23:17.5)HTU24:18.5)HTU21:17.0)HTU22:50.5)HTU16[717.0];
(AlbanianTosk:67.0,((ArmenianWestern:25.0,(Assamese:30.5,SanskritVedic:15.0)HTU23:28.0)HTU24:22.5,(Scots:28.5,(((AncientIonicGreek:42.5,Irish:31.5)HTU19:13.0,((Cornish:26.0,Manx:24.0)HTU18:23.0,(Danish:27.0,(OldPrussian:27.5,(Faroese:39.5,Icelandic:53.0)HTU29:19.5)HTU30:14.0)HTU27:17.5)HTU28:19.5)HTU20:17.5,(Yiddish:30.5,(EasternFrisianSaterland:33.5,KolschColognian:47.5)HTU17:16.0)HTU21:19.0)HTU22:18.5)HTU25:17.0)HTU26:50.5)HTU16[717.0];
(AlbanianTosk:65.5,(Scots:30.5,((ArmenianWestern:22.5,(Assamese:30.5,SanskritVedic:15.0)HTU18:29.0)HTU19:14.5,((Yiddish:32.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU17:17.5)HTU20:20.5,((AncientIonicGreek:45.5,Irish:28.5)HTU29:27.5,((Cornish:26.5,Manx:23.5)HTU27:21.0,(Danish:27.5,(OldPrussian:29.5,(Faroese:25.5,Icelandic:58.5)HTU25:22.5)HTU26:12.0)HTU28:15.0)HTU30:17.5)HTU23:17.5)HTU24:14.0)HTU21:14.5)HTU22:43.0)HTU16[717.0];
(AlbanianTosk:65.5,(Scots:30.5,((ArmenianWestern:22.5,(Assamese:30.5,SanskritVedic:15.0)HTU22:29.0)HTU23:14.5,(((AncientIonicGreek:45.5,Irish:28.5)HTU19:27.5,((Cornish:26.5,Manx:23.5)HTU18:21.0,(Danish:27.5,(OldPrussian:29.5,(Faroese:25.5,Icelandic:58.5)HTU29:22.5)HTU30:12.0)HTU27:15.0)HTU28:17.5)HTU20:17.5,(Yiddish:32.0,(EasternFrisianSaterland:34.0,KolschColognian:47.0)HTU17:17.5)HTU21:20.5)HTU24:14.0)HTU25:14.5)HTU26:43.0)HTU16[717.0];
