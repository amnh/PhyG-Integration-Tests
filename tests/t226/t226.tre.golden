(AlbanianTosk:49.5,(Danish:41.5,(((OldPrussian:32.5,(Faroese:25.5,Icelandic:58.5)HTU21:18.5)HTU20:11.0,(((Cornish:29.5,Manx:20.5)HTU24:24.5,(Yiddish:33.0,(EasternFrisianSaterland:33.5,KolschColognian:47.5)HTU26:15.5)HTU25:23.0)HTU23:18.0,(Assamese:43.5,Scots:38.5)HTU27:19.5)HTU22:16.5)HTU19:25.0,(ArmenianWestern:28.5,(SanskritVedic:29.0,(AncientIonicGreek:42.0,Irish:32.0)HTU30:30.0)HTU29:23.5)HTU28:21.0)HTU18:17.0)HTU17:27.0)HTU16[774.0];
(AlbanianTosk:66.5,(Danish:38.5,(((OldPrussian:33.0,(Faroese:26.5,Icelandic:57.5)HTU21:20.0)HTU20:12.0,(((Cornish:29.0,Manx:21.0)HTU24:25.5,(EasternFrisianSaterland:27.0,Yiddish:44.0)HTU25:18.0)HTU23:17.0,(Assamese:53.5,Scots:32.5)HTU26:16.0)HTU22:15.5)HTU19:12.0,(ArmenianWestern:27.0,(KolschColognian:50.0,(SanskritVedic:31.5,(AncientIonicGreek:41.0,Irish:33.0)HTU30:22.0)HTU29:22.5)HTU28:23.0)HTU27:20.5)HTU18:18.5)HTU17:52.0)HTU16[773.0];
(AlbanianTosk:50.5,(Danish:42.0,(((OldPrussian:33.5,(Faroese:25.0,Icelandic:59.0)HTU21:19.5)HTU20:13.0,(((Irish:25.5,(Cornish:27.0,Manx:23.0)HTU25:25.5)HTU24:19.0,(Yiddish:31.0,(EasternFrisianSaterland:33.0,KolschColognian:48.0)HTU27:16.5)HTU26:19.5)HTU23:20.0,(Assamese:42.5,Scots:39.5)HTU28:16.5)HTU22:16.5)HTU19:21.0,(ArmenianWestern:29.0,(AncientIonicGreek:50.5,SanskritVedic:23.5)HTU30:24.0)HTU29:19.0)HTU18:21.0)HTU17:26.5)HTU16[769.0];
